myriad-scheduler/src/test/java/com/ebay/myriad/scheduler/SchedulerUtilsSpec.groovy

```diff
@@ -29,6 +30,30 @@ class SchedulerUtilsSpec extends Specification {
 
     }
 
+    def "is eligible for FGS"() {
+        given:
+        def state = Mock(SchedulerState)
+        def tasks = new ArrayList<NodeTask>()
+        def fgsNMTask = new NodeTask(new NMProfile("zero", 0, 0))
+        def cgsNMTask = new NodeTask(new NMProfile("low", 2, 4096))
+        fgsNMTask.setHostname("test_fgs_hostname");
```

Comment:
groovy doesn't need semi colons :)  also it is more idiomatic to do `fgsNMTask.hostname ="test_fgs_hostname"`